WordPress/src/main/res/layout/my_site_post_card_with_post_items.xml

```diff
@@ -28,5 +31,15 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/my_site_toolbar" />
 
+        <include
+            android:id="@+id/my_site_card_footer_link"
+            layout="@layout/my_site_card_footer_link"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_small"
```

Comment:
Question (❓): Why is the `android:layout_marginTop` needed here?

Reply:
This top margin is added to accommodate the extra `4dp` in the layouts for the post cards `with` and `without` post items as per the specs (highlighted in red).

<img width="384" alt="footer_link_margin" src="https://user-images.githubusercontent.com/1405144/142147519-266d1e5b-66be-4415-a4ca-a90dfc43eefa.png">

Reply:
👋 @ashiagr !

Ah, I see, I see, thanks for the explanation and for the visual input! 🌟